netContain/contivNet.sh

```diff
@@ -0,0 +1,13 @@
+#!/bin/bash
+#Initialize complete contiv container. Start OVS and Net Plugin
+
+if [ $1 == "reinit" ]; then
+    ovs-vsctl del-br contivVlanBridge
+    ovs-vsctl del-br contivVxlanBridge
+fi
+
+/contiv/scripts/ovsInit.sh
+/contiv/scripts/contivInit.sh
```

Comment:
Should we look into using supervisors(http://supervisord.org) instead of just running these processes in the background?
Problem is, if netplugin or netmaster die because of a bug, there is no way to restart it